test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj

```diff
@@ -23,6 +23,21 @@
     </ItemGroup>
     <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\CSharp\%(RecursiveDir)" />
     <Message Text="Copied Samples output to $(OutDir)\SqlExtensionSamples" Importance="high" />
+    <ItemGroup>
+      <_CopyItems Include="..\samples\samples-js\**\*.*" />
+    </ItemGroup>
+    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\JavaScript\%(RecursiveDir)" />
```

Comment:
Did you verify that this isn't going to copy the csharp stuff in here too? I'm not sure offhand whether _CopyItems will be overwritten each time, but just to be safe (and clearer) I'd suggest making each of these have a different name (so `_CSharpCopyItems`, `_JsCopyItems`, etc.) 

And you should only need to have 1 Copy task for each (so 1 for CSharp, 1 for JS, etc.). This should work 

```
    <ItemGroup>
      <_JsCopyItems Include="host.json" />
      <_JsCopyItems Include="..\samples\samples-js\**\*.*" />
      <_JsCopyItems Include="Integration/test-js\**\*.*" />
    </ItemGroup>
    <Copy SourceFiles="@(_JsCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\JavaScript\" />
```